plugins/org.polarsys.capella.scenario.editor.doc/html/Release Notes/Whatsnew.html

```diff
@@ -2,7 +2,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 	<head>
 		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
-		<title>What's New</title>
```

Comment:
keep What's New

Reply:
unfornatelty the title comes from the file name. I guess there was an edition by hand